sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_compiler.asciidoc

```diff
@@ -618,6 +618,150 @@ property, but the duplicates have no effect.
 _{endnote}_]
 |====
 
+=== New free function for linking
+
+This extension adds the following new free functions to create and build a
+kernel bundle in `ext_oneapi_source` state.
```

Comment:
This introductory sentence seems wrong.  These functions do not "create" or "build" kernel bundles, and they are not limited to bundles in the `ext_oneapi_source` state.  It seems like we should say something like this instead:

> This extension adds the following new functions to link kernel bundles while also passing properties that affect the semantics of the link operation.